src/test/java/io/strimzi/ValidatingWebhookTest.java

```diff
@@ -78,6 +78,7 @@ public void testV1Beta1Eviction() {
                 .withNewMetadata()
                     .withName(podName)
                     .withNamespace("my-namespace")
+                    .withLabels(Collections.singletonMap("strimzi.io/name", "my-cluster-kafka"))
```

Comment:
I would expect to see tests without any labels, with some completely different label, with correct label with wrong value and finally with correct label and correct Kafka and ZooKeeper value.

In addition, a test with Kafka label but disabled Kafka draining and ZooKeeper label and disabled ZooKeeper draining would be good.